### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by liyasthomas**
- Use template literals.
- Prefer Object destructuring assignment approach.
- `let` to `const`.
- Other minor tweaks.

### CODE REVIEW

File: components/ace-editor.vue [hunk 1]
```diff
@@ -74,13 +74,10 @@ export default {
...
-        return this.theme;
-      } else {
-        return (
-          this.$store.state.postwoman.settings.THEME_ACE_EDITOR || DEFAULT_THEME
-        );
-      }
+      if (this.theme) return this.theme;
+      return (
+        this.$store.state.postwoman.settings.THEME_ACE_EDITOR || DEFAULT_THEME
+      );
```
Comment (jamesgeorge007) [inline components/ace-editor.vue]:
```js
if () {
  //
}
return ...
```

Having a consistent style across the codebase is really important. What do you think?

File: pages/index.vue [hunk 2]
```diff
@@ -1885,54 +1883,33 @@ export default {
       if (this.requestType === "JavaScript XHR") {
         const requestString = [];
         requestString.push("const xhr = new XMLHttpRequest()");
-        const user =
-          this.auth === "Basic Auth" ? "'" + this.httpUser + "'" : null;
+        const user = this.auth === "Basic Auth" ? `'${this.httpUser}'` : null;
         const pswd =
-          this.auth === "Basic Auth" ? "'" + this.httpPassword + "'" : null;
+          this.auth === "Basic Auth" ? `'${this.httpPassword}'` : null;
```
Comment (jamesgeorge007) [inline pages/index.vue]: Ig we can go with the `if/else` approach here. Also regarding the names, `pswd` doesn't sound that good.

File: pages/index.vue [hunk 3]
```diff
@@ -2295,9 +2264,8 @@ export default {
       }));
     },
     pathInputHandler() {
-      let queryString = this.getQueryStringFromPath(),
-        params = this.queryStringToArray(queryString);
-
+      let queryString = this.getQueryStringFromPath();
```
Comment (jamesgeorge007) [inline pages/index.vue]:
```suggestion
      const queryString = this.getQueryStringFromPath();
```

File: pages/index.vue [hunk 4]
```diff
@@ -2295,9 +2264,8 @@ export default {
...
+      let params = this.queryStringToArray(queryString);
```
Comment (jamesgeorge007) [inline pages/index.vue]:
```suggestion
      const params = this.queryStringToArray(queryString);
```

File: pages/index.vue (hunk 3 above)
Comment (liyasthomas) [reply pages/index.vue]: `queryString`'s scope is only required inside that block, so let's go wil `let`

**Pull request merged**